build-scripts/aur-git/50-systemd-user.conf

```diff
@@ -12,5 +12,5 @@ exec systemctl --user import-environment DISPLAY \
 exec hash dbus-update-activation-environment 2>/dev/null && \
      dbus-update-activation-environment --systemd DISPLAY \
                                                   SWAYSOCK \
-                                                  XDG_CURRENT_DESKTOP=sway \
+                                                  XDG_CURRENT_DESKTOP="swayfx:sway" \
```

Comment:
Q: does this env var play well with having multiple values?

Reply:
Yep. Erik asked the same question (see previous review).